nixos/tests/filesystems-overlayfs.nix

```diff
@@ -26,7 +26,6 @@ in
 
   nodes.machine = { config, pkgs, ... }: {
     boot.initrd.systemd.enable = true;
-    boot.initrd.availableKernelModules = [ "overlay" ];
```

Comment:
Help me understand why this needed to leave? Is it just excess to requirements now?

Reply:
It's just redundant is all. It never needed to be there in the first place.